src/hearings/services/judicial-ref-data.service.ts

```diff
@@ -11,4 +11,9 @@ export class JudicialRefDataService {
     return this.http.post<JudicialUserModel[]>('api/prd/judicial/searchJudicialUserByPersonalCodes',
       { personal_code: personalCodes });
   }
+
+  public searchJudicialUserByIdamID(personalCodes: string[]): Observable<JudicialUserModel[]> {
```

Comment:
this function is confused. personal codes are not idam ids - they are a completely different identifier. Check what you are passing to make sure that it actually is an idam ID, and rename the parameter

Reply:
Yeah this should have been renamed to idamIds, have updated